app/models/user.rb

```diff
@@ -0,0 +1,2 @@
+class User < ApplicationRecord
```

Comment:
Let's not forget to set up these associations and validations

Reply:
Good call.